### PR TITLE
[FIX] 10n_fr_fec : fec can be download by unalowaed user

### PR DESCRIPTION
**Pull request by fmdl**
Before this commit a user from an other company can download (by rpc) a fec created by an other user.

@odony 
@oco-odoo 
@alexis-via 
@Yenthe666 

---
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Comment by fmdl** [comment]
OPW #3240374

**Comment by fmdl** [comment]
ping @oco-odoo

File: addons/l10n_fr_fec/security/security.xml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo noupdate="1">
+    <record id="account_fr_fec_rule" model="ir.rule">
+        <field name="name">Account Fr Fec Rule</field>
+        <field name="model_id" ref="model_account_fr_fec"/>
+        <field name="domain_force">[('create_uid', '=', user.id)]</field>
+    </record>
+</odoo>
```
Comment (aliyatastemirova) [inline addons/l10n_fr_fec/security/security.xml:8]: A small thing: can you please add a newline at the end of this file?

**Review by aliyatastemirova — CHANGES_REQUESTED** [review]
Could you please write a clearer commit message as well? Also, I think it would be nicer to avoid typos (should be `can be downloaded by unallowed user` for example)

**Review by oco-odoo — CHANGES_REQUESTED** [review]
Please squash your commits. And while you're at it, also fix the typo in the title. Then, I merge :)

**Comment by fmdl** [comment]
ping @oco-odoo

**Review by oco-odoo — APPROVED** [review]
@robodoo r+

**Pull request closed without merging**